pi_ldapproxy/proxy.py

```diff
@@ -255,7 +258,18 @@ class ProxyServerFactory(protocol.ServerFactory):
     def __init__(self, config):
         # NOTE: ServerFactory.__init__ does not exist?
         # Read configuration options.
-        self.agent = Agent(reactor)
+        if config['privacyidea']['verify']:
+            if config['privacyidea']['certificate']:
+                certificate = Certificate.loadPEM(FilePath(config['privacyidea']['certificate']).getContent())
+                log.info('Pinning privacyIDEA HTTPS certificate {certificate!r}', certificate=certificate)
```

Comment:
Does this logentry looks readable at all? The representation for the certificate? Is the filename of the certificate contained? Maybe in the logfile it is better, to know from **where** the certificate was loaded?